src/gatsby-ssr.js

```diff
@@ -40,28 +41,34 @@ exports.onRenderBody = ({ setHeadComponents }, pluginOptions) => {
   // note below, snippet wont render unless writeKey is truthy
   const writeKey = process.env.NODE_ENV === "production" ? prodKey : devKey;
 
-  const loadConfig = controlPlaneUrl
-    ? `'${writeKey}', '${dataPlaneUrl}', {configUrl: '${controlPlaneUrl}'}`
-    : `'${writeKey}', '${dataPlaneUrl}'`;
-  /*
-    if (controlPlaneUrl) {
-      return `'${writeKey}', '${dataPlaneUrl}', {configUrl: '${controlPlaneUrl}'}`;
-    } else {
-      return `'${writeKey}', '${dataPlaneUrl}'`;
-    }
-  }; */
+  if (controlPlaneUrl) {
+    loadOptions.configUrl = controlPlaneUrl;
+  }
```

Comment:
We should never modify the inputs.

Reply:
That is how it is being done in gatsby. So, I didn't change that functionality.

Reply:
That's incorrect.

Reply:
Addressed